src/mon/Paxos.cc

```diff
@@ -77,39 +77,39 @@ void Paxos::init()
+  pcb.add_u64_avg(l_paxos_begin_bytes, "begin_bytes", "Data in transaction on begin");
+  pcb.add_time_avg(l_paxos_begin_latency, "begin_latency", "Latency of begin operation");
   pcb.add_u64_counter(l_paxos_commit, "commit",
       "Commits", "cmt");
-  pcb.add_u64_avg(l_paxos_commit_keys, "commit_keys");
-  pcb.add_u64_avg(l_paxos_commit_bytes, "commit_bytes");
+  pcb.add_u64_avg(l_paxos_commit_keys, "commit_keys", "Keys in transaction on commit");
+  pcb.add_u64_avg(l_paxos_commit_bytes, "commit_bytes", "Data in transaction on commit");
   pcb.add_time_avg(l_paxos_commit_latency, "commit_latency",
       "Commit latency", "clat");
-  pcb.add_u64_counter(l_paxos_collect, "collect");
-  pcb.add_u64_avg(l_paxos_collect_keys, "collect_keys");
-  pcb.add_u64_avg(l_paxos_collect_bytes, "collect_bytes");
-  pcb.add_time_avg(l_paxos_collect_latency, "collect_latency");
-  pcb.add_u64_counter(l_paxos_collect_uncommitted, "collect_uncommitted");
-  pcb.add_u64_counter(l_paxos_collect_timeout, "collect_timeout");
-  pcb.add_u64_counter(l_paxos_accept_timeout, "accept_timeout");
-  pcb.add_u64_counter(l_paxos_lease_ack_timeout, "lease_ack_timeout");
-  pcb.add_u64_counter(l_paxos_lease_timeout, "lease_timeout");
-  pcb.add_u64_counter(l_paxos_store_state, "store_state");
-  pcb.add_u64_avg(l_paxos_store_state_keys, "store_state_keys");
-  pcb.add_u64_avg(l_paxos_store_state_bytes, "store_state_bytes");
-  pcb.add_time_avg(l_paxos_store_state_latency, "store_state_latency");
-  pcb.add_u64_counter(l_paxos_share_state, "share_state");
-  pcb.add_u64_avg(l_paxos_share_state_keys, "share_state_keys");
-  pcb.add_u64_avg(l_paxos_share_state_bytes, "share_state_bytes");
-  pcb.add_u64_counter(l_paxos_new_pn, "new_pn");
-  pcb.add_time_avg(l_paxos_new_pn_latency, "new_pn_latency");
+  pcb.add_u64_counter(l_paxos_collect, "collect", "Peon collects");
+  pcb.add_u64_avg(l_paxos_collect_keys, "collect_keys", "Keys in transaction on peon collect");
+  pcb.add_u64_avg(l_paxos_collect_bytes, "collect_bytes", "Data in transaction on peon collect");
+  pcb.add_time_avg(l_paxos_collect_latency, "collect_latency", "Peon collect latency");
+  pcb.add_u64_counter(l_paxos_collect_uncommitted, "collect_uncommitted", "Uncommitted values in started and handled collects");
+  pcb.add_u64_counter(l_paxos_collect_timeout, "collect_timeout", "Collect timeouts");
+  pcb.add_u64_counter(l_paxos_accept_timeout, "accept_timeout", "Accept timeouts");
+  pcb.add_u64_counter(l_paxos_lease_ack_timeout, "lease_ack_timeout", "Lease acknowledgement timeouts");
+  pcb.add_u64_counter(l_paxos_lease_timeout, "lease_timeout", "Lease timeouts");
+  pcb.add_u64_counter(l_paxos_store_state, "store_state", "Store a shared state on disk");
+  pcb.add_u64_avg(l_paxos_store_state_keys, "store_state_keys", "Keys in transaction on storing state");
+  pcb.add_u64_avg(l_paxos_store_state_bytes, "store_state_bytes", "Data in transaction on storing state");
```

Comment:
s/on storing state/in stored state/

As it is phrased it seems like 'storing' (same for 'sharing' below) is a Paxos state -- which in this case is not and we can easily avoid the mix up :)